spec/support/acceptance/examples.rb

```diff
@@ -1,4 +1,4 @@
-shared_examples 'the foreman application' do
+shared_examples 'the foreman application' do |params = { login_url: '/users/login' }|
```

Comment:
I wonder about this. What if a we add more parameters, but a user doesn't specify a login_url. I don't think they'll get merged.

Instead I'd suggest to use `params = {}` and then `params.get(:login_url, '/users/login')`
 in the actual test. I think it's cleaner than storing a hash somewhere and merging them. Discoverability may be an issue, but I think it's not really an issue right now. We're not making a full API here.

Also a slight naming nit: I'd name this a path, not a URL. Perhaps even `login_url_path`. And if we're fully making it self describing I'd add `expected` to make it `expected_login_url_path`.